ml-dhs-aws/src/main/java/com/marklogic/client/dhs/ObjectLoader.java

```diff
@@ -34,6 +37,8 @@ public void loadRecords(Iterator<ObjectNode> objectItr) {
    public void loadRecord(ObjectNode record) {
       String uri = directory+UUID.randomUUID().toString()+".json";
       count++;
+      record.set("instance", this.recordDef);
+      record.set("metadata", this.recordMetadata);
```

Comment:
Let's create a new ObjectNode after line 39 and set the "instance" property of the new object node to record and the "metadata" property of the new object node to recordMetadata.  That way, the new ObjectNode becomes a wrapper for the record converted from a CSV row and the metadata provided with the job.